src/components/cards/Card.module.css

```diff
@@ -1,7 +1,7 @@
-.collectiontitle a:focus {
+.cardtitle a:focus {
```

Comment:
Nit: use camel case (or maybe camel case for JS and kebab for HTML?)

Reply:
Update: this repo uses kebab for html attribute values but in this case it should probably be camel case for ease of use in the component file.